### PR TITLE
Modify documents of flags in advanced_guide

### PR DESCRIPTION
**Pull request by gfwm0502**
修改进阶指南中`环境变量FLAGS`的中英文文档，添加使用介绍部分。

180 76 141 178_documentation_docs_en_advanced_guide_flags_flags_en html (2)
180 76 141 178_documentation_docs_zh_advanced_guide_flags_flags_cn html (1)

### CODE REVIEW

File: doc/fluid/advanced_guide/flags/flags_en.rst [hunk 1]
```diff
@@ -2,6 +2,17 @@
 FLAGS
 ==================
 
+Method of use
```
Comment (zhhsplendid) [inline doc/fluid/advanced_guide/flags/flags_en.rst]: Method of use -> Usage

**Review by zhhsplendid — APPROVED** [review]
Other thing LGTM

File: doc/fluid/advanced_guide/flags/flags_en.rst (hunk 1 above)
Comment (gfwm0502) [reply doc/fluid/advanced_guide/flags/flags_en.rst]: Thanks，done。

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Review by swtkiwi — APPROVED** [review]
LGTM

File: doc/fluid/advanced_guide/flags/flags_cn.rst [hunk 2]
```diff
@@ -2,11 +2,22 @@
 环境变量FLAGS
 ==================
 
+调用说明
+----------
+
+PaddlePaddle中的环境变量FLAGS支持两种设置方式。
+
+- 通过export来设置环境变量，如 :code:`export FLAGS_eager_delete_tensor_gb = True` 。
```
Comment (zhiqiu) [inline doc/fluid/advanced_guide/flags/flags_cn.rst]: The value of `FLAGS_eager_delete_tensor_gb` should be double, like 1.0 .

**Review by Heeenrrry — APPROVED** [review]
LGTM

File: doc/fluid/advanced_guide/flags/flags_cn.rst (hunk 2 above)
Comment (gfwm0502) [reply doc/fluid/advanced_guide/flags/flags_cn.rst]: Thanks， done.

**Pull request merged**